### ISSUE TITLE
Allow restricting live validation to a subset of rows

### ISSUE THREAD

**Issue by maxxxxxdlp** [specify/specify7#852]
For example, validate the selected rows first when the "Data Check" button is pressed.
Should be implemented after #851 to make it obvious which rows have been / would be validated.

**Comment by maxxxxxdlp** [comment]
This is a wontfix since we decided to disable live validation

**Comment by grantfitzsimmons** [comment]
Max, we need to add live validation!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by maxxxxxdlp** [comment]
It's already there. Just need to bring back the button

**Comment by maxxxxxdlp** [comment]
Which color should the button be? 😂

**Comment by grantfitzsimmons** [comment]
Theresa says we should use orange. It's her favorite color

**Comment by grantfitzsimmons** [comment]
Almost 1 year later, we need to add it back
